src/Views/Wallpaper.vala

```diff
@@ -106,6 +112,15 @@ public class PantheonShell.Wallpaper : Gtk.Grid {
         var add_wallpaper_button = new Gtk.Button.with_label (_("Import Photo…"));
         add_wallpaper_button.margin = 12;
 
+        var dim_label = new Gtk.Label ("Dim wallpaper in dark style:") {
```

Comment:
As "dim" can be either a verb or an adjective maybe "Dim the wallpaper in dark style:" would be easier to translate? (Or add a ///TRANSLATORS comment)